victoriametrics.yaml

```diff
@@ -13,13 +13,31 @@ data:
       vmalert: vmalert executes a list of the given alerting or recording rules against configured -datasource.url compatible with Prometheus HTTP API.
       vmauth: vmauth is a simple auth proxy and router for VictoriaMetrics
 
+environment:
+  contents:
+    packages:
+      - nodejs-20
+      - npm
+
 pipeline:
   - uses: git-checkout
     with:
       expected-commit: 123f373537323f4e7278db5601dab4e67759af1c
       repository: https://github.com/VictoriaMetrics/VictoriaMetrics
       tag: v${{package.version}}
 
+  - runs: |
+      cp docs/victoriametrics/MetricsQL.md app/vmui/packages/vmui/src/assets/MetricsQL.md
```

Comment:
If we intend to copy this to the main package then we should create `app/vmui/packages/vmui/src/assets/` directory and cp into `${{targets.contextdir}}` 

or this is expected this way only, as of now, this seems build-time only.

Reply:
so here's me understanding of this:
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/app/vmui/README.md#static-build I am getting the static build files in the right place i.e. under `app/` so when we finally build vm its embedded.

In the steps added, I am replicating the same things (the make targets) in our build. We dont need them in the final binary.

Reply:
thank you for the references, cross verified it again and this looks good.